annotate.el

```diff
@@ -115,6 +115,43 @@
   :type 'string
   :group 'annotate)
 
+(defcustom annotate-maximum-size-checksum 5000000
+  "Calculate checksum of  the current buffer only if  the size is
+less than this size (in characters)"
+  :type  'integer
+  :group 'annotate)
+
+(defconst annotate-warn-file-changed-control-string
+  (concat "The file '%s' has changed on disk "
+          "from the last time the annotations was saved. "
+          "Chances are that they will not be displayed correctly")
+  "The  message to  warn user  that  file has  been modified  and
+  annototatnions position could be outdated")
```

Comment:
...the annotations *were* saved...
...to warn *the* user that *the* file has been modified and *annotation positions* could...

Reply:
Thank you for fix my bad english!